src/Glpi/Console/CommandLoader.php

```diff
@@ -368,8 +369,7 @@ private function findSymfonyCommands(): void
             return;
         }
 
-        if (!\in_array($kernel->getEnvironment(), [\GLPI::ENV_DEVELOPMENT, \GLPI::ENV_TESTING], true)) {
-            // Don't load commands if non-dev/test
+        if (!Environment::get()->shouldLoadSymfonyCommands()) {
```

Comment:
Right now, we load them only for debugging.

```suggestion
        if (!Environment::get()->shouldEnableExtraDevAndDebugTools()) {
```